modules/programs/qutebrowser.nix

```
@@ -260,16 +260,20 @@ in {
     };
   };
 
-  config = mkIf cfg.enable {
+  config = let
+    qutebrowserConfig = if pkgs.stdenv.hostPlatform.isDarwin then
+      ".qutebrowser/config.py"
+    else
+      ".config/qutebrowser/config.py";
+  in mkIf cfg.enable {
     home.packages = [ cfg.package ];
 
-    xdg.configFile."qutebrowser/config.py".text = concatStringsSep "\n" ([ ]
-      ++ [
-        "${if cfg.loadAutoconfig then
-          "config.load_autoconfig()"
-        else
-          "config.load_autoconfig(False)"}"
-      ] ++ mapAttrsToList (formatLine "c.") cfg.settings
+    home.file."${qutebrowserConfig}".text = concatStringsSep "\n" ([ ] ++ [
+      "${if cfg.loadAutoconfig then
+        "config.load_autoconfig()"
+      else
+        "config.load_autoconfig(False)"}"
+    ] ++ mapAttrsToList (formatLine "c.") cfg.settings
```

```suggestion
    home.file."${qutebrowserConfig}".text = concatStringsSep "\n" ([
      (if cfg.loadAutoconfig then
        "config.load_autoconfig()"
      else
        "config.load_autoconfig(False)")
    ] ++ mapAttrsToList (formatLine "c.") cfg.settings
```

The expression can be simplified to be a bit cleaner.